cg/meta/upload/scout/balsamic_config_builder.py

```diff
@@ -57,15 +57,8 @@ def build_config_sample(self, case_sample: FamilySample) -> ScoutCancerIndividua
         self.add_common_sample_files(config_sample=config_sample, case_sample=case_sample)
         if BalsamicAnalysisAPI.get_sample_type(sample_obj=case_sample.sample) == SampleType.TUMOR:
             config_sample.phenotype = PhenotypeStatus.AFFECTED.value
-            config_sample.sample_id = SampleType.TUMOR.value.upper()
         else:
             config_sample.phenotype = PhenotypeStatus.UNAFFECTED.value
```

Comment:
```
config_sample.phenotype = PhenotypeStatus.AFFECTED.value if sample_type == SampleType.TUMOR else PhenotypeStatus.UNAFFECTED.value
```

Reply:
I prefer the original way actually 😬 I think it's very clear. Otherwise I'd also need to add this line:
`sample_type = BalsamicAnalysisAPI.get_sample_type(sample_obj=case_sample.sample)`

Reply:
👍